include/envoy/ratelimit/ratelimit.h

```diff
@@ -24,6 +24,10 @@ struct RateLimitOverride {
 struct DescriptorEntry {
   std::string key_;
   std::string value_;
+
+  friend bool operator==(const DescriptorEntry& lhs, const DescriptorEntry& rhs) {
+    return lhs.key_ == rhs.key_ && lhs.value_ == rhs.value_;
+  }
```

Comment:
Is this needed? You have no hash function here so probably not?

Reply:
Removed